index.js

```diff
@@ -1 +1,7 @@
-require('./built').default();
+const fs = require('fs');
+
+if (fs.existsSync('./built')) {
+	require('./built').default();
+} else {
+	console.log('Built code is not found. Probably an error occurred during a build or you just forgot to build.');
```

Comment:
stdout に入れられるのは違和感なのと、そもそも終了コードが正常終了のそれになってしまう。

```suggestion
	console.error('Built code is not found. Probably an error occurred during a build or you just forgot to build.');
	process.exit(1);
```

Reply:
CC: @AyaMorisawa